### PR TITLE
Test

### PR DESCRIPTION
**Pull request by danny237**

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+For test branch version 2.0
```
Comment (silwalanish) [inline test.txt]: Now I can suggest you changes if required. And this is how code review works.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# git-tutorial
+This is awesome! 
+
```
Comment (silwalanish) [inline README.md]: For example: I could as you to insert the date here.
@danny237 Can you add today's date here?

Comment (danny237) [reply README.md]: Sure Bor!!

**Comment by silwalanish** [comment]
Also, It is rarely that someone who opens the PR merges the PR themselves.

**Pull request merged**